src/app/form-email/form-email.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {FormEmailComponent} from './form-email.component';
+
+describe('FormEmailComponent', () => {
+  let component: FormEmailComponent;
+  let fixture: ComponentFixture<FormEmailComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [FormEmailComponent],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(FormEmailComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
i test dovrebbero esser un po' più espressivi; testare che viene creato e basta non così utile credo..

Reply:
Questi test li crea in automatico angular quando generiamo un component, mi informerò se hanno davvero poca utilità

Reply:
ah allora forse conviene modificare la schematic di angular per velocizzare il processo. comunque piuttosto che toBeTruthy ho visto che jasmine ha il metodo toBeDefined che è più espressivo